Prox/Prox/Utilities/AppConstants.swift

```diff
@@ -13,7 +13,9 @@ public enum AppBuildChannel {
 public struct AppConstants {
 
     public static let isRunningTest = NSClassFromString("XCTestCase") != nil
-    public static let backgroundFetchInterval: TimeInterval = 5 * 60
+    public static let backgroundFetchInterval: TimeInterval = 1 * 60
+    public static let minimumTimeAtLocationBeforeFetchingEvents: TimeInterval = 1 * 60
```

Comment:
Do we want to commit 60s updates for all builds? Perhaps we should separate this depending on local, BB, etc.